nixos/modules/security/sudo.nix

```diff
@@ -247,7 +245,7 @@ in
       };
     };
 
-    environment.systemPackages = [ sudo ];
+    environment.systemPackages = [ cfg.package ];
```

Comment:
Considering we have the SUID wrappers, is sudo in `systemPackages` needed at all?

Reply:
At the very least for man-pages & localized messages, perhaps for sudoers.so et al., but that's enough reason already to not discuss it in this bugfix PR. If you'd like to investigate further, feel free :)